src/components/Greeting.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
```

Comment:
- [ ] Kindly make sure you install the following `react-redux`, `react-router-dom`, `@reduxjs/toolkit` before using it, it is displaying errors on the page. Check the package.json file to see all installed packages

Reply:
Thanks